en/flight_controller/cuav_v5_nano.md

```diff
@@ -140,61 +140,44 @@ The *V5 nano* has no over current protection.
-<!-- Nano/80/V5_NANO_RC01_2019_05_29 -->
-
-- *Found:* Batch 01
-- *Fixed:* -
+CUAV adopts some differentiated designs and is incompatible with some hardware, which will be described below.
 
-#### PM2 cannot power the flight controller {#issue_pm2}
+#### Neo v2.0 GPS not compatible with other devices {#compatibility_gps}
 
-`PM2` can only measure battery voltage and current, but **not** power the flight controller.
-
-> **Warning** We recommend you do not connect a power module to this connector when using PX4.
+The *Neo v2.0 GPS* that is recommended for use with *CUAV V5+* and *CUAV V5 nano* is not fully compatible with other Pixhawk flight controllers (specifically, the buzzer part is not compatible and there may be issues with the safety switch).
 
-- *Found:* Batch 01
-- *Fixed:* -
+UAVCAN GPS is ideal for avoiding this compatibility.
 
-#### Volt regulation varies greater than +/- 5%
+#### Using JTAG for hardware debugging {#compatibility_jtag}
 
-The 5 volt pins on all connectors will be lower when powered from USB than the Power Module (the pins will measure approximately 4.69V when only powered by USB, and 5.28 Volts when connected to the Power Module).
-
-We recommend that when using USB with the *V5 nano* you *also connect the power module* (to avoid under-powering any connected peripherals).
+`DSU7` FMU Debug Pin 1 is 5 volts - not the 3.3 volts of the CPU.
 
-> **Warning** Remove propellers *before* connecting the power module (this is important whenever bench testing with powered motors).
+Some JTAG probes use this voltage to set the IO levels when communicating to the target.
 
-- *Found:* Batch 01
-- *Fixed:* -
+For direct connection to *Segger Jlink* we recommended you use the 3.3 Volts of DSM/SBUS/RSSI pin 4 as Pin 1 on the debug connector (`Vtref`).
 
+#### PM2 cannot power the flight controller {compatibility#_pm2}
 
-#### Using JTAG for hardware debugging {#issue_jtag}
+`PM2` can only measure battery voltage and current, but **not** power the flight controller.
 
-`DSU7` FMU Debug Pin 1 is 5 volts - not the 3.3 volts of the CPU.
+> **Warning** PX4 does not support this interface.
 
-Some JTAG use this voltage to set the IO levels when communicating to the target.
+## Known Issues
 
-For direct connection to *Segger Jlink* we recommended you use the 3.3 Volts of `DSM`/`SBUS`/`RSSI` pin 4 as Pin 1 on the debug connector (`Vtref`).
+The issues below refer to the *batch number* in which they first appear. 
+The batch number is the four-digit production date behind V01 and is displayed on a sticker on the side of the flight controller.
+For example, the serial number Batch V011904((V01 is the number of V5, 1904 is the production date, that is, the batch number).
 
-#### HV\_PM power module output is unfused {#issue_pm_unfused}
+#### SBUS / DSM / RSSI interface Pin1 unfused {#pin1_unfused}
 
-> **Warning** This is a serious safety issue.
+> **Warning** This is a security issue. 
```

Comment:
@cuhome What is a security issue? Are you saying that the power module output is fine and there is no risk to health here?

Reply:
You are right, it should be a safety issue.